src/Agent.Sdk/Util/LoggedSecretMasker.cs

```diff
@@ -6,31 +6,24 @@ namespace Agent.Sdk.Util
     /// <summary>
     /// Extended secret masker service, that allows to log origins of secrets
     /// </summary>
-    public class LoggedSecretMasker : ILoggedSecretMasker
+    public class LoggedSecretMasker : SecretMasker, ILoggedSecretMasker
```

Comment:
>LoggedSecretMasker [](http://example.com/codeflow?start=17&length=18)

Rather than wrapping the secret masked defined in the TFS assembly, we now extend the ported `SecretMasker` class. By doing so, we can still reference & use interface contracts (such as `ISecretMasker`) that are still defined in the TFS binary. This is required because other code in this repository needs to continue to use these TFS-defined contracts. In particular, `IExpressionNode` requires the TFS `ISecretMasker` interface. Resolving this dependency would entail pulling over lots of code, it appears.